vars/promoteArtifacts.groovy

```diff
@@ -32,22 +37,47 @@ void call(Map args = [:]) {
             String pluginSubFolder = pluginSubPath.split('/')[0]
             String pluginNameWithExt = pluginSubPath.split('/')[1]
             String pluginName = pluginNameWithExt.replace('-' + version + '.zip', '')
+            String pluginNameNoExt = pluginNameWithExt.replace('-' + version, '')
             String pluginFullPath = ['plugins', pluginName, version].join('/')
-            s3Upload(bucket: "${ARTIFACT_PRODUCTION_BUCKET_NAME}", path: "releases/$pluginFullPath/", workingDir: "$WORKSPACE/artifacts/$artifactPath/builds/$filename/$pluginSubFolder/"
-                , includePathPattern: "**/${pluginName}*")
+            for (Closure action : fileActions) {
+                for (file in findFiles(glob: "$WORKSPACE/artifacts/$artifactPath/builds/$filename/$pluginSubFolder/**/${pluginName}*")) {
```

Comment:
We can merge here but as #1355 points out we need to save these params into their own var.

Reply:
This PR doesn't resolve #1355, but it did some of the things called out there so I figured it was better to mention it.